examples/ilql_sentiments.py

```diff
@@ -36,7 +36,6 @@ def metric_fn(samples: List[str]) -> Dict[str, List[float]]:
     imdb = load_dataset("imdb", split="train+test")
 
     trlx.train(
-        "gpt2",
```

Comment:
I like this change. Previously, if your ILQL config set the `model_path` option it would be overridden by this `"gpt2"` arg. It's led to unexpected behavior on my end whereby editing the example ILQL config's model path doesn't actually update the model.

Reply:
Yes, it's for consistency with other examples